Makefile

```diff
@@ -29,6 +29,7 @@ include env.d/development
 
 BOLD := \033[1m
 RESET := \033[0m
+GREEN := \033[36m
```

Comment:
```suggestion
GREEN := \033[1;32m
```
To me it was blue not green, I much prefer this one!

Reply:
Yes it was `cyan` and not green. Using a brighter color is intended ?

Reply:
I just extracted the initial color you used, which is cyan, my bad.
We should stick with color names matching codes https://www.shellhacks.com/bash-colors/ as displayed color depend on local terminal settings.